pkg/prometheus/asset_store_test.go

```diff
@@ -26,6 +26,45 @@ import (
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 )
 
+var (
+	caPEM = `-----BEGIN CERTIFICATE-----
```

Comment:
It seems that this certificate is only valid until `Oct  8 12:52:00 2021 GMT`. Can you extend its lifetime to something very far in the future?

Reply:
Thank you for checking, I've added a hundred years to it.